config/config.json

```diff
@@ -17,219 +17,18 @@
-        "misc-unused-parameters" : ["extreme"], 
-        "misc-unused-raii" : ["default", "sensitive", "extreme"],
-        "misc-unused-using-decls" : ["sensitive", "extreme"],
-        "misc-use-after-move" : ["default", "sensitive", "extreme"],
-        "misc-virtual-near-miss" : ["default", "sensitive", "extreme"],
-        "modernize-deprecated-headers" : ["sensitive", "extreme"],
-        "modernize-replace-auto-ptr" : ["sensitive", "extreme"],
-        "modernize-replace-random-shuffle" : ["sensitive", "extreme"],
-        "mpi-buffer-deref" : ["sensitive", "extreme"],
-        "mpi-type-mismatch" : ["sensitive", "extreme"],
-        "performance-faster-string-find" : ["sensitive", "extreme"],
-        "performance-for-range-copy" : ["sensitive", "extreme"],
-        "performance-implicit-conversion-in-loop" : ["sensitive", "extreme"],
-        "performance-inefficient-algorithm" : ["default", "sensitive", "extreme"],
-        "performance-inefficient-string-concatenation" : ["sensitive", "extreme"],
-        "performance-inefficient-vector-operation" : ["sensitive", "extreme"],
-        "performance-move-const-arg" : ["default", "sensitive", "extreme"],
-        "performance-noexcept-move-constructor" : ["default", "sensitive", "extreme"],
-        "performance-type-promotion-in-math-fn" : ["sensitive", "extreme"],
-        "performance-unnecessary-copy-initialization" : ["sensitive", "extreme"],
-        "performance-unnecessary-value-param" : ["sensitive", "extreme"]
+          "default": "-e bugprone-assert -e bugprone-bool -e bugprone-copy -e bugprone-dangling -e bugprone-fold -e bugprone-forward -e bugprone-inaccurate -e bugprone-incorrect -e bugprone-integer -e bugprone-lambda -e bugprone-macro-repeated -e bugprone-misplaced -e bugprone-move -e bugprone-sizeof -e bugprone-string-compare -e bugprone-string-constructor -e bugprone-string-literal -e bugprone-suspicious-enum -e bugprone-suspicious-memset -e bugprone-suspicious-missing -e bugprone-suspicious-semicolon -e bugprone-swapped -e bugprone-terminating -e bugprone-throw -e bugprone-too -e bugprone-undefined -e bugprone-undelegated -e bugprone-unused -e bugprone-use -e bugprone-virtual -e cert-dcl54 -e cert-dcl58 -e cert-dcl59 -e cert-err09 -e cert-fio38 -e cert-oop11 -e cppcoreguidelines-special -e google-build-namespaces -e google-global -e misc-assert -e misc-bool -e misc-dangling -e misc-definitions -e misc-fold -e misc-forward -e misc-forwarding -e misc-inaccurate -e misc-incorrect -e misc-inefficient -e misc-lambda -e misc-macro -e misc-misplaced -e misc-move -e misc-redundant -e misc-sizeof -e misc-string-compare -e misc-string-constructor -e misc-string-literal -e misc-suspicious-enum -e misc-suspicious-missing -e misc-suspicious-semicolon -e misc-swapped -e misc-unconventional -e misc-uniqueptr -e misc-unused-raii -e misc-use -e misc-virtual -e performance-inefficient-algorithm -e performance-move -e performance-noexcept",
```

Comment:
I'm thinking about a small change to convert this to a list:
```json
   "default": [
    "-e bugprone-assert",
    "-e bugprone-bool",
    "-e bugprone-copy",
    "-e bugprone-dangling",
    "-e bugprone-fold"
  ]
```
I think reading and parsing it would be easier, the same applies for the other profiles.

Reply:
I have created an enable-only list, and I have updated the PR to reflect the most recent changes.